### PR TITLE
Add config.json file

### PR DESCRIPTION
**Pull request by rishabhja**

### CODE REVIEW

File: modules/swagger-codegen/src/main/resources/Java/config.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"modelPackage" : "co.flock.model",
```
Comment (prateeksultania) [inline modules/swagger-codegen/src/main/resources/Java/config.json]: fix indentation

File: modules/swagger-codegen/src/main/resources/Javascript/config.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+	"modelPackage" : "model",
```
Comment (prateeksultania) [inline modules/swagger-codegen/src/main/resources/Javascript/config.json]: fix indentation

File: modules/swagger-codegen/src/main/resources/Java/config.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+ 	"eventPackage" : "co.flock.event",
+    "apiPackage" : "co.flock.api",
+    "groupId" : "co.flock",
+    "artifactId" : "flockos-java-client"
```
Comment (prateeksultania) [inline modules/swagger-codegen/src/main/resources/Java/config.json]: Also add the option "hideGenerationTimestamp" : true

File: modules/swagger-codegen/src/main/resources/Java/config.json (hunk 1 above)
Comment (rishabhja) [inline modules/swagger-codegen/src/main/resources/Java/config.json]: done

File: modules/swagger-codegen/src/main/resources/Java/config.json (hunk 3 above)
Comment (rishabhja) [inline modules/swagger-codegen/src/main/resources/Java/config.json]: done

**Pull request merged**